3-mcstas/SANS_from_function.ipynb

```diff
@@ -206,11 +218,20 @@
     ")"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "moved-injury",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "quiz.question_2(instrument)"
```

Comment:
I don't know if it's expected, but this yieds an error
![Screenshot at 2023-09-03 08-11-18](https://github.com/ess-dmsc-dram/dmsc-school/assets/39047984/92a64259-63a5-46c6-8706-7ee6d60cfe4e)

Reply:
This is surprising, it accepts the answer locally and 1.5 should be the correct answer.

Reply:
See the fix here? https://github.com/ess-dmsc-dram/dmsc-school/pull/41/files#diff-3d2a02b1302f9885c0cc6216e4b0af27721aeac8cbc123617055a516ff0a94a6R48

Reply:
You had 6 for wavelength and 6 for `d_wavelength`